python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -6,9 +6,31 @@
 import torch
 
 from ..base import etype_str_to_tuple
+from ..itemset import ItemSet, ItemSetDict
 from ..minibatch import MiniBatch
 
 
+def full_nodes_itemset(
```

Comment:
What about support range here https://github.com/dmlc/dgl/blob/c298223f5de09e9ee265cf6a9fb9145b692b4c5b/python/dgl/graphbolt/itemset.py#L66?

And then provide a util function to handle dict?